src/Akka.Streams.Kafka/Stages/ConsumerStage.cs

```diff
@@ -68,6 +71,12 @@ public KafkaSourceStageLogic(KafkaSourceStage<K, V, Msg> stage, Attributes attri
                 }
                 else
                 {
+                    if (isPaused)
```

Comment:
If a downstream ask for an item and we were paused and have an empty buffer - just resume consuming messages from Kafka